SafeExamBrowser.SystemComponents/VirtualMachineDetector.cs

```diff
@@ -8,14 +8,18 @@
 
 using SafeExamBrowser.Logging.Contracts;
 using SafeExamBrowser.SystemComponents.Contracts;
+using System.Globalization;
+using System.Linq;
+using System.Management;
+using System.Net.NetworkInformation;
 
 namespace SafeExamBrowser.SystemComponents
 {
 	public class VirtualMachineDetector : IVirtualMachineDetector
 	{
 		private ILogger logger;
 		private ISystemInfo systemInfo;
-
+		private static readonly string[] pciVendorBlacklist = { "vbox", "80ee", "qemu", "1af4", "1b36" }; //Virtualbox: VBOX, 80EE   RedHat: QUEMU, 1AF4, 1B36
```

Comment:
And please move the declaration of this array to the top of the class with an empty line after it and use CAPS_NAMING as it is immutable.